modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/PortletBase.es.js

```diff
@@ -24,18 +22,39 @@ function toElementHelper(elementOrSelector) {
 	return elementOrSelector;
 }
 
-function isString(val) {
-	return typeof val === 'string';
+class LifeCycles {
+	constructor() {
+		this.created();
+
+		this.attached();
+	}
+	dispose() {
+		this.disposeInternal();
+
+		this.detached();
+
+		this.disposed();
+	}
+	attached() {}
```

Comment:
Hey @bryceosterhaus, why did you leave these methods there? I think it was for testing purposes, right?

Reply:
```
 * The purpose of this class is to mimic the lifecycles from metal-state.
 * There are several classes that extend PortletBase and expect some of these
 * lifecycles to exist. This pseudo-lifecycle class is meant to provide backwards
 * compatibility for classes extending PortletBase.
```

My bad, closing this comment 🙈

Reply:
I think it may have been a github UI bug, because it didn't show this comment to me until I cleared my cache, even though I saw it locally and committed it 🤷